src/Module/Following.php

```diff
@@ -14,7 +14,7 @@
  */
 class Following extends BaseModule
 {
-	public static function init()
+	public static function rawContent()
```

Comment:
Can you add the method to `BaseModule` please?

Reply:
Okay.